### PR TITLE
Geany.Geany-Plugins version 1.38

### PR DESCRIPTION
**Pull request by mariobrostech**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/51558)

### CODE REVIEW

**Comment by mariobrostech** [comment]
I was able to successfully test the package installation using the following commands on my local machine:
`winget validate --manifest manifests/g/Geany/Geany-Plugins/1.38`
`winget install --manifest manifests/g/Geany/Geany-Plugins/1.38`

I'm not sure why the bot is reporting a failed installation. Can anyone explain what might be going wrong? I know this installer requires the `/S` argument to install silently, but winget seems to be automatically using this when running on my system.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image
image

`WinGet-2022-03-11-18-09-52.575.log`:
```
2022-03-11 18:09:52.576 [CORE] WinGet, version [1.3.431-preview], activity [{46C395C0-BAE8-468A-9446-BF480C05A1D4}]
2022-03-11 18:09:52.577 [CORE] OS: Windows.Desktop v10.0.22572.1
2022-03-11 18:09:52.577 [CORE] Command line Args: "C:\Users\WDAGUtilityAccount\AppData\Local\Microsoft\WindowsApps\winget.exe" install --manifest C:\Users\WDAGUtilityAccount\Documents\GitHub\winget-pkgs\manifests\g\Geany\Geany-Plugins\1.38
2022-03-11 18:09:52.577 [CORE] Package: Microsoft.DesktopAppInstaller v1.18.431.0
2022-03-11 18:09:52.577 [CORE] IsCOMCall:0; Caller: winget-cli
2022-03-11 18:09:52.585 [CLI ] WinGet invoked with arguments: 'install' '--manifest' 'C:\Users\WDAGUtilityAccount\Documents\GitHub\winget-pkgs\manifests\g\Geany\Geany-Plugins\1.38'
2022-03-11 18:09:52.585 [CLI ] Found subcommand: install
2022-03-11 18:09:52.585 [CLI ] Leaf command to execute: root:install
2022-03-11 18:09:52.588 [CLI ] Executing command: install
2022-03-11 18:09:52.589 [CLI ] Manifest fields: Name [Geany Plugins], Version [1.38]
2022-03-11 18:09:52.592 [CLI ] Starting installer selection.
2022-03-11 18:09:52.592 [CLI ] Completed installer selection.
2022-03-11 18:09:52.611 [CLI ] Generated temp download path: C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\3bb95f8aed7e37de495b899865d9fc51b951e7385842f2c11df2cdddbd5f80f6
2022-03-11 18:09:52.612 [CORE] Downloading to path: C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\3bb95f8aed7e37de495b899865d9fc51b951e7385842f2c11df2cdddbd5f80f6
2022-03-11 18:09:52.612 [CORE] Started applying motw to C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\3bb95f8aed7e37de495b899865d9fc51b951e7385842f2c11df2cdddbd5f80f6 with zone: 3
2022-03-11 18:09:52.616 [CORE] Finished applying motw
2022-03-11 18:09:52.617 [CORE] WinINet downloading from url: https://plugins.geany.org/geany-plugins/geany-plugins-1.38_setup.exe
2022-03-11 18:09:54.292 [CORE] Download hash: 3bb95f8aed7e37de495b899865d9fc51b951e7385842f2c11df2cdddbd5f80f6
2022-03-11 18:09:54.292 [CORE] Download completed.
2022-03-11 18:09:54.293 [CLI ] Installer hash verified
2022-03-11 18:09:54.293 [CORE] Started applying motw using IAttachmentExecute to C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\3bb95f8aed7e37de495b899865d9fc51b951e7385842f2c11df2cdddbd5f80f6
2022-03-11 18:09:55.272 [CORE] Finished applying motw using IAttachmentExecute. Result: 0 IAttachmentExecute::Save() result: 0
2022-03-11 18:09:55.273 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DownloadFlow.cpp(141)\WindowsPackageManager.dll!00007FFE0A4E5295: (caller: 00007FFE0A344881) LogHr(1) tid(17c8) 8007023E {Application Error}

The exception %s (0x    Msg:[std::exception: rename: The system cannot find the file specified.: "C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\3bb95f8aed7e37de495b899865d9fc51b951e7385842f2c11df2cdddbd5f80f6", "C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\geany-plugins-1.38_setup.exe"] 

2022-03-11 18:09:55.274 [CLI ] Successfully renamed downloaded installer. Path: C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\geany-plugins-1.38_setup.exe
2022-03-11 18:09:55.289 [REPO] Source named 'msstore' is already defined at origin User. The source from origin Default is dropped.
2022-03-11 18:09:55.289 [REPO] Creating PredefinedInstalledSource with filter [ARP]
2022-03-11 18:09:55.289 [REPO] Creating new SQLite Index [4294967295.4294967295] at ':memory:'
2022-03-11 18:09:55.289 [SQL ] Opening SQLite connection: ':memory:' [6, 0]
2022-03-11 18:09:55.340 [REPO] Examining ARP entries for Machine | X64
2022-03-11 18:09:55.340 [REPO] Examining ARP entries for Machine | X86
2022-03-11 18:09:55.340 [REPO] Examining ARP entries for User | X64
2022-03-11 18:09:55.394 [CLI ] Installer args: /S
2022-03-11 18:09:55.395 [CLI ] Starting: 'C:\Users\WDAGUT~1\AppData\Local\Temp\WinGet\Geany.Geany-Plugins.1.38\geany-plugins-1.38_setup.exe' with arguments '/S'
2022-03-11 18:09:57.068 [CLI ] ShellExecute installer failed: 2
2022-03-11 18:09:57.069 [CLI ] Terminating context: 0x8a150006 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:162
```

Looks like it has a dependency on Geany.

**Comment by mariobrostech** [comment]
That would explain it. I didn't know that Geany has to be installed prior to installing the plugins, but that would make sense. Is there any way I can specify dependencies in the manifest, or will this issue need to be held until [#163](https://github.com/microsoft/winget-cli/issues/163) is implemented?

File: manifests/g/Geany/Geany-Plugins/1.38/Geany.Geany-Plugins.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: Geany.Geany-Plugins
+PackageVersion: 1.38
+PackageLocale: en-US
+Publisher: The Geany contributors
+PublisherUrl: https://www.geany.org/
+PublisherSupportUrl: https://github.com/geany/geany-plugins/issues
+PrivacyUrl: https://geany.org/about/privacy/
+Author: The Geany contributors
+PackageName: Geany Plugins
```
Comment (ItzLevvie) [inline manifests/g/Geany/Geany-Plugins/1.38/Geany.Geany-Plugins.locale.en-US.yaml:12]:
```suggestion
PackageName: Geany-Plugins
```
This must match Control Panel.

File: manifests/g/Geany/Geany-Plugins/1.38/Geany.Geany-Plugins.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Geany.Geany-Plugins
+PackageVersion: 1.38
+PackageLocale: en-US
+Publisher: The Geany contributors
```
Comment (ItzLevvie) [inline manifests/g/Geany/Geany-Plugins/1.38/Geany.Geany-Plugins.locale.en-US.yaml:7]:
```suggestion
Publisher: The Geany developer team
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by mariobrostech** [comment]
@ItzLevvie I’ve merged your suggestions into my latest commit.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**